cli/lsp/analysis.rs

```diff
@@ -388,12 +396,15 @@ pub fn analyze_dependencies(
       {
         dep.maybe_type_specifier_range = Some(Range {
           start: Position {
-            line: (loc.line - 1) as u32,
-            character: (loc.col + 1) as u32,
+            line: loc.line_index as u32,
+            // +1 to skip quote
```

Comment:
I'm not sure I understand what this comment means

Reply:
The `+ 1` on the next line is to skip over the quote of the string literal. Do you have a suggestion for a better comment?

Reply:
Okay then in case of `end` shouldn't it be `-1` to skip the quote? Otherwise it will include the quote, won't it? Maybe I misunderstood something then just ignore the comment :)

Reply:
Sorry I didn't comment it originally, but feels like we need a test here, as there was so much manipulation... Yeah we are trying to ignore the quotes around the string literal, but it brings into question what is happening on line 406.

Reply:
No, `loc.column_index + 1` is the start and then it's `start + specifier.chars().count()` to get the end. What I'll do is extract out `loc.column_index + 1` to a common variable (but will do that tomorrow).

Reply:
Oh, wait, I see what you mean. I think `specifier` doesn't include the quotes, but `loc` does IIRC. I will investigate it tomorrow.

Reply:
Considering this code hasn't changed and that we're going to replace this logic with what's in deno_graph, I'm just going to resolve this comment and merge.